deployment/environment/memory/node.go

```diff
@@ -466,10 +472,12 @@ func CreateKeys(t *testing.T,
 			}
 			transmitters[chain.Selector] = transmitter.String()
 
-			backend := chain.Client.(*Backend).Sim
-			fundAddress(t, chain.DeployerKey, transmitter, assets.Ether(1000).ToInt(), backend)
-			// need to look more into it, but it seems like with sim chains nodes are sending txs with 0x from address
-			fundAddress(t, chain.DeployerKey, common.Address{}, assets.Ether(1000).ToInt(), backend)
+			backend, ok := chain.Client.(*Backend)
+			if ok {
```

Comment:
In which cases is this `ok` not `true`?

Reply:
It is `true` for memory environment. We can flag it in a different way. For ZK this is `false`